docs/streams/stream-examples/realtime-updates.md

```diff
@@ -42,17 +49,41 @@ const data = [
 const sleep = (milliseconds) => {
   return new Promise(resolve => setTimeout(resolve, milliseconds));
 };
+```
+
+</TabItem>
 
-// Create an authenticated instance with a token or API key
+<TabItem value="py" label="Python">
 
-// const client = new jsc8({url: globalUrl, token: "XXXX", fabricName: '_system'});
-// const client = new jsc8({url: globalUrl, apiKey: "XXXX", fabricName: '_system'});
+```py
+COLLECTION_NAME = "ddos"
 
-// Or use email and password to authenticate client instance
+# Variables - Data
+data = [
+    {"ip": "10.1.1.1", "action": "block", "rule": "blacklistA"},
+    {"ip": "20.1.1.2", "action": "block", "rule": "blacklistA"},
+    {"ip": "30.1.1.3", "action": "block", "rule": "blacklistB"},
+    {"ip": "40.1.1.4", "action": "block", "rule": "blacklistA"},
+    {"ip": "50.1.1.5", "action": "block", "rule": "blacklistB"},
+]
 
+pp = pprint.PrettyPrinter(indent=4)
```

Comment:
I got this error after adding the second step. Please be sure to thoroughly test your documentation, both the full demo and the step-by-step versions.

```
(base) dianapayton@Dianas-MacBook-Pro python % python real-time.py
--- Connecting to GDN
Traceback (most recent call last):
  File "/Users/dianapayton/Documents/python/real-time.py", line 33, in <module>
    pp = pprint.PrettyPrinter(indent=4)
NameError: name 'pprint' is not defined
```